test/Jamfile.v2

```diff
@@ -490,6 +490,7 @@ test-suite special_fun :
    [ run test_trig.cpp test_instances//test_instances pch_light ../../test/build//boost_unit_test_framework  ]
    [ run test_zeta.cpp ../../test/build//boost_unit_test_framework test_instances//test_instances pch_light  ]
    [ run test_sinc.cpp ../../test/build//boost_unit_test_framework pch_light ]
+   [ run test_fibonacci.cpp ../../test/build//boost_unit_test_framework : : : <linkflags>-lgmp [ requires cpp_lambdas ] ]
```

Comment:
I don't really require lamdba, but it's a hack for C++11 check required by `<cstdint>`

Reply:
@madhur4127: I personally don't mind if this is written in C++17. I'm doing all my new features in C++17 so that if I need to make changes that require C++17 I won't break anything.

Hopefully that's not being obtuse.